samples/apps/copilot-chat-app/webapi/CopilotChat/Extensions/SemanticKernelExtensions.cs

```diff
@@ -65,8 +65,12 @@ private static KernelBuilder WithPlannerBackend(this KernelBuilder kernelBuilder
     {
         return options.Type switch
         {
-            AIServiceOptions.AIServiceType.AzureOpenAI => kernelBuilder.WithAzureChatCompletionService(options.Models.Planner, options.Endpoint, options.Key),
-            AIServiceOptions.AIServiceType.OpenAI => kernelBuilder.WithOpenAIChatCompletionService(options.Models.Planner, options.Key),
+            AIServiceOptions.AIServiceType.AzureOpenAI => kernelBuilder.WithAzureChatCompletionService(options.Models.Planner,
+                (string.IsNullOrWhiteSpace(options.Models.PlannerEndpoint)) ? options.Endpoint : options.Models.PlannerEndpoint,
```

Comment:
```suggestion
                (string.IsNullOrWhiteSpace(options.PlannerOverrides.Endpoint)) ? options.Endpoint : options.PlannerOverrides.PlannerEndpoint,
```

Reply:
Need to update the rest as well

Reply:
@thomasgobin - let us know if you need anything here or if you have any questions on the changes.